insights/tests/test_yaml_parser.py

```diff
@@ -18,12 +26,13 @@ class MyYamlParser(YAMLParser):
     pass
 
 
-def test_json_parser_success():
-    ctx = context_wrap('a: 2')
-    assert FakeYamlParser(ctx)
+def test_yaml_parser_success():
+    for ymlstr in yaml_test_strings:
+        ctx = context_wrap(ymlstr)
+        assert FakeYamlParser(ctx)
 
 
-def test_json_parser_failure():
+def test_yaml_parser_failure():
     ctx = context_wrap("boom /")
     with pytest.raises(ParseException) as ex:
         FakeYamlParser(ctx)
```

Comment:
@fcami your changes look good.  Would you mind to add some additional tests for the contents of the parser to ensure that they are properly converted regardless of whether its a list or a dictionary?

Reply:
hi @bfahr this was done in another commit (add test for when YAML test is a list) where 'a:2' was used (this results in a dictionary). I've expanded that with a larger dictionary in commit https://github.com/RedHatInsights/insights-core/pull/2061/commits/d0fc662ce2eb9709d5cb2bd3f75876e812c6f941 now.